### PR TITLE
Move victim name to the right of people table

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18816

## Testing
**URL to test:** 
[Netlify](https://deploy-preview-1532--atd-vze-staging.netlify.app/)

**Steps to test:**
1. For people with fatal injuries, see that the victim name column is now to the far right of the table. Check this for primary and non primary people.

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

**Review by johnclary — CHANGES_REQUESTED** [review]
this looks good, but there is a little horizontal scrollbar that seems to be visible at every page width. can you squash whatever is tripping that?

Screenshot 2024-08-28 at 12 04 33 PM

File: atd-vze/src/views/Crashes/VictimNameField.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const VictimNameField = ({
   return (
     <td>
       {isEditing && (
-        <Form>
```
Comment (johnclary) [inline atd-vze/src/views/Crashes/VictimNameField.js:43]: can we keep the form element here, too? just as a best practice.

**Review by johnclary — APPROVED** [review]
horizontal scrollbar is gone! would you mind adding the `<form>` back in? then ship it! 🚢 🚢 🚢 🚢

**Review by chiaberry — APPROVED** [review]
✅

**Pull request merged**